rskj-core/src/main/java/co/rsk/crypto/Keccak256.java

```diff
@@ -18,36 +18,54 @@
 
 package co.rsk.crypto;
```

Comment:
As the wrapper this is, it sounds weird to put on crypto. Looks something for commons or related. There's nothing about crypto here but the name of the class.

Reply:
Agreed. Would you think that co.rsk.core would be more apropiate, or should i create a commons package? If so, i would also would have to include RskAddress, don't you think?

Reply:
.common's cool to me. I'll comment it on #programming and we see if someone opposes to it